src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,58 @@
+.watch {
+  border: $size-border solid greenyellow;
+  border-radius: 50%;
+
+  &__dot {
+    &::after {
+      position: absolute;
+      top: $position-dot;
+      left: $position-dot;
+      z-index: 2;
+
+      height: $size-dot;
+      width: $size-dot;
+
+      content: "";
+
+      background-color: rgb(0, 0, 0);
+      border-radius: 50%;
+    }
+  }
+
+  &__minutes {
+    &::after {
+      position: absolute;
+      left: $size-watch / 2 - $width-minutes / 2;
+      top: $size-watch / 2 - $height-minutes;
+      z-index: 1;
+
+      height: $height-minutes;
+      width: $width-minutes;
+
+      content: "";
+
+      background-color: rgb(70, 136, 108);
+      border-radius: 10px;
+
+      animation: move 60s * 60 infinite steps(60);
+      transform-origin: bottom center;
+    }
+  }
+
+  &__seconds {
+    &::after {
+      position: absolute;
+      left: $height-seconds - $witdh-seconds / 2;
+
+      content: "";
+      height: $height-seconds;
+      width: $witdh-seconds;
+
+      background-color: rgb(136, 202, 139);
+      border-radius: 10px;
+
+      animation: move 60s infinite steps(60);
+      transform-origin: bottom center;
+    }
+  }
```

Comment:
You can use here native elements, `after` is unnecessary here

Reply:
agree, try to move these styles to &__seconds instead of `after`